Modules/_randommodule.c

```diff
@@ -259,16 +259,27 @@ random_seed(RandomObject *self, PyObject *args)
      * So: if the arg is a PyLong, use its absolute value.
      * Otherwise use its hash value, cast to unsigned.
      */
-    if (PyLong_Check(arg))
+    if (PyLong_Check(arg)) {
         n = PyNumber_Absolute(arg);
+        if (n == NULL) {
+            goto Done;
+        }
+        if (!PyLong_Check(n)) {
+            PyErr_Format(PyExc_TypeError,
+                         "abs(a) must return an integer, not '%.200s'",
```

Comment:
`abs(seed)`?

Reply:
i agree this seems weird, but that's what the docs say (https://docs.python.org/3.7/library/random.html#random.seed).
should i change to `abs(seed)` anyway?

Reply:
or should we update the docs instead?

Reply:
The doc is about `random.Random.seed()`. This is `_random.Random.seed()`. It doesn't have named arguments. I don't know what is the best wording, but mentioning `a` whose name is arbitrary and not related to this function looks incorrect to me.

I leave up this to @rhettinger.